### PR TITLE
Improve .post-create message

### PR DESCRIPTION
**Pull request by lorensr**
Based on this user support question: https://temporalio.slack.com/archives/CTRCR8RBP/p1647015491342459

If we like this, I'll propagate to sample directories before merging.

### CODE REVIEW

**Review by bergundy — COMMENTED** [review]
Why is this Mac specific?

**Comment by lorensr** [comment]
Idea is to make it simplest for those that need clearest instructions. My sense from JS ecosystem is that most devs, regardless of their OS, would be used to seeing `brew install` instructions, and know to do the equivalent for their OS. 

I could certainly change to links to download pages tho. @sw-yx thoughts?

**Comment by cretz** [comment]
IMO we don't need to tell people how to install docker, and especially shouldn't put OS-specific instructions without acknowledging it's for a specific OS. Even the `cd ~/temporal-server` is platform specific.

**Comment by vkarpov15** [comment]
I don't like the idea of putting OSX-specific instructions, because it implies that this only works on OSX.

**Comment by nelsonic** [comment]
Just chiming as a recent adopter of Temporal and trying to on-board a team of engineers ... 
Agree that having MacOS-specific instructions could give the impression that it's MacOS-only. 
However _not_ having _any_ instructions means some people get stuck and have to Google
i.e. increase friction / barriers to adoption.
It might be easier to just add a line: 
This works on all platforms. If you're not using MacOS, Help us by extending these docs with instructions for your OS.

**Comment by bergundy** [comment]
I'm fine with *adding* MacOS specific instructions and possibly customizing the post create message based on the OS (although the latter a bit overkill and I wouldn't say it is high priority now).

File: .shared/.post-create [hunk 1]
```diff
@@ -1,10 +1,18 @@
-To begin development, start Temporal Server:
+To begin development, install Docker:
 
-  {cyan cd} ~/path/to/temporal/docker-compose/
+  {cyan brew cask install docker}
+  {cyan open} /Applications/Docker.app
+  {cyan brew install docker-compose}
```
Comment (sw-yx) [inline .shared/.post-create:5]: i have a dumb question. how expensive would it be to check if docker is already installed, and/or if docker daemon is already running? feel like i'd like to get rid of this msg if its unnecessary.

**Review by sw-yx — APPROVED** [review]
left a comment but lgtm otherwise

File: .shared/.post-create (hunk 1 above)
Comment (lorensr) [reply .shared/.post-create:5]: Don't know but good idea! Added:

https://github.com/temporalio/sdk-typescript/issues/541

**Pull request merged**